_health-care/_js/utils/validations.js

```diff
@@ -12,6 +12,14 @@ function isValidDate(day, month, year) {
     date.getFullYear() === Number(year);
 }
 
+function isValidName(value) {
+  return value === '' || /^[a-zA-Z '\-]+$/.test(value);
+}
+
+function isValidRequired(value) {
```

Comment:
I needed a way to distinguish between name fields that are required and name fields that aren't (e.g., veteran name is required, spouse is not). I didn't want to include check this in the `isValidName` function because it seemed like a separate requirement that could be useful for other validations as well, so I put this in its own function to be used in combination with `isValidName` for required name fields.